docs/tutorials/python/json_schema.md

```diff
@@ -21,16 +21,16 @@ By the end of this tutorial, you will:
 * You are familiar with [adding annotations](./annotation.md) to synapse entity.
```

Comment:
nit: Would also add that familiarity with `SchemaOrganization` is also necessary

Reply:
You would need a schema organization to work with jsonschema so some routes...

1. Merge the two into one doc (just call it jsonschemas)
2. Add that familiarity is required
3. Flesh out the section

It may be easier to do option 1, but i'm open to other suggestions.

Reply:
@andrewelamb Actually, I think we can add more details on the sections below if needed about schema organizations since this tutorial is about jsonschemas.  

I just read the purpose "2. Learn how to retrieve an organization and schema. If they do not exist, understand how to create them."  so i think is it covered.  We can resolve this nit